catalog/app/components/Dropzone/Dropzone.js

```diff
@@ -0,0 +1,100 @@
+import cx from 'classnames'
+import * as React from 'react'
+import { useDropzone } from 'react-dropzone'
+import * as M from '@material-ui/core'
+
+import DropMessage from './DropMessage'
+import FileEntry from './FileEntry'
+import Header from './Header'
+
+const useStyles = M.makeStyles((t) => ({
+  dropzone: {
+    marginTop: t.spacing(2),
+    position: 'relative',
+  },
+  dropArea: {
+    background: t.palette.action.hover,
+    border: `1px solid ${t.palette.action.disabled}`,
+    borderRadius: t.shape.borderRadius,
+    cursor: 'pointer',
+    display: 'flex',
+    flexDirection: 'column',
+    minHeight: 140,
+    outline: 'none',
+    overflow: 'hidden',
+  },
+  dragOver: {
+    background: t.palette.action.selected,
+  },
+  warning: {
+    borderColor: t.palette.warning.dark,
+  },
+  error: {
+    borderColor: t.palette.error.main,
+  },
+  files: {
+    borderBottom: `1px solid ${t.palette.action.disabled}`,
+    maxHeight: 200,
+    overflowX: 'hidden',
+    overflowY: 'auto',
+  },
+}))
+
+export default function Dropzone({
+  className,
+  disabled,
+  error,
+  files,
+  onDrop,
+  overlayComponent,
+  statsComponent,
```

Comment:
What do you propose for a name then? `statsInstance`?

Reply:
widely used convention is `Element` suffix, e.g. `statsElement`. in case of this particular components, maybe we can omit this altogether and use just `stats` and `overlay`